game-core/src/main/java/swinglib/JLabelBuilder.java

```diff
@@ -50,15 +48,7 @@ public static JLabelBuilder builder() {
   public JLabel build() {
     Preconditions.checkState(text != null || icon != null);
 
-    final String truncated;
-    if (text != null) {
-      Preconditions.checkState(!text.trim().isEmpty());
-      truncated = Ascii.truncate(text, maxTextLength, "...");
-    } else {
-      truncated = "";
-    }
-
-    final JLabel label = new JLabel(truncated);
+    final JLabel label = new JLabel(text);
```

Comment:
I'm curious to know what lead to this change in behaviour.

Reply:
I do not quite understand your comment @RoiEXLab. If you're asking why this change, the `truncated` value is always the same as `text`, the 'truncation' feature is unused code.

Reply:
Ah I missed that `maxTextLength` was `Integer.MAX_VALUE`